src/Components/DnDLayout/GridTile.tsx

```diff
@@ -153,7 +158,10 @@ const GridTile = ({ widgetType, isDragging, setIsDragging, setWidgetAttribute, w
       </Tooltip>
       <Tooltip aria-label="Move widget" content={<p>{widgetConfig.static ? 'Widget locked' : 'Move'}</p>}>
         <Icon
-          onMouseDown={() => setIsDragging(true)}
+          onMouseDown={() => {
+            setIsDragging(true);
+            analytics.track('widget-move', { widgetTitle });
+          }}
```

Comment:
@florkbr It seems to trigger this event every time the user clicks on the move icon on the widgets. We might need to find something else for the analytics to listen to?

Reply:
@tahmidefaz I think for now (based on the Jira ticket) we just want to know if a widget it being moved and if so which ones. I think putting the listener on the drag handle works for now, and then we will need to investigate more in the future around:
1. Which widget appears first in the layout?
2. When a widget is moved, where is it moved from/to?